src/FunctionApps/python/tests/IntakePipeline/pipeline_test.py

```diff
@@ -6,48 +6,64 @@
 TEST_ENV = {
     "INTAKE_CONTAINER_URL": "some-url",
     "INTAKE_CONTAINER_PREFIX": "some-prefix",
-    "OUTPUT_CONTAINER_PATH": "output/path",
+    "VALID_OUTPUT_CONTAINER_PATH": "output/valid/path",
+    "INVALID_OUTPUT_CONTAINER_PATH": "output/invalid/path",
     "HASH_SALT": "super-secret-definitely-legit-passphrase",
     "SMARTYSTREETS_AUTH_ID": "smarty-auth-id",
     "SMARTYSTREETS_AUTH_TOKEN": "smarty-auth-token",
     "FHIR_URL": "fhir-url",
 }
 
+MESSAGE_MAPPINGS = {
+    "bundle_type": "VXU",
+    "root_template": "VXU_V04",
+    "input_data_type": "Hl7v2",
+    "template_collection": "microsofthealth/fhirconverter:default",
+    "filename": "some-filename-1",
+}
+
 
-@mock.patch("IntakePipeline.read_fhir_bundles")
 @mock.patch("IntakePipeline.transform_bundle")
 @mock.patch("IntakePipeline.add_patient_identifier")
 @mock.patch("IntakePipeline.upload_bundle_to_fhir_server")
-@mock.patch("IntakePipeline.store_bundle")
-@mock.patch("IntakePipeline.get_fhirserver_cred_manager")
+@mock.patch("IntakePipeline.store_data")
 @mock.patch("IntakePipeline.get_smartystreets_client")
+@mock.patch("IntakePipeline.convert_message_to_fhir")
 @mock.patch.dict("os.environ", TEST_ENV)
```

Comment:
this is getting to be a *lot*

Reply:
I think you're both right.  I feel like having a higher level orchestration component (eg. data factory) might be a better architectural direction.  This is OK for the pilot, but the more blocks we add to this, the more unwieldy it becomes to manage, and further away from intended use of a azure function.  We should foster this kind of thinking to apply post-pilot.

Reply:
@spencer-skylight Actually the goal for us is to provide a lot of different, plug and play independent blocks that STLTs can pick from at their leisure. It's actually a strength of our service that we're making so many things independent because it helps us deliver mission value, even if that does mean the organization gets kind of wonky.